conftest.py

```diff
@@ -7,18 +7,13 @@
 
 # Standard Library
 import functools
-import copy
 
 # Third Party Stuff
 import pytest
 import django
 from django.conf import settings
 
-from tests import test_settings
-
-backends = ["twilio.TwilioBackend", "nexmo.NexmoBackend"]
-sandbox_backends = ["twilio.TwilioSandboxBackend", "nexmo.NexmoSandboxBackend"]
-all_backends = backends + sandbox_backends
+from tests.fixtures import backend
```

Comment:
Why do you need this import?

Reply:
So that, I don't have to import the fixture `backend` in any other test module. By importing it within `conftest.py`, it'll automatically be available to all the other test functions.